### PR TITLE
feat(replay): Updated weekly email to optionally include Total Project Replays section

### PR DESCRIPTION
**Pull request by ryan953**
Relates to https://github.com/getsentry/sentry/issues/51369

Sharing early with some debugging code still in here, and unimplemented spots too

You can see example emails by running `sentry devserver` and visiting http://dev.getsentry.net:8000/debug/mail/weekly-reports/

There are 6 ui cases that can happen

Note that the 'setup replays' copy in the table below is not final. These screens are to test padding/borders and layout.

The real copy is looking like (still need the Play image added):
SCR-20230622-odyf

| total_transaction_count | has_replay_section #L742 | total_replay_count | | layout | txn view? | replay view? | img |
| --- | --- | --- | --- | --- | --- | --- | --- |
| 0 | False | n/a | | two-up | setup | n/a |  setup txn |
| 0 | True | 0 | | three-up | setup | setup | setup txn - setup replay 
| 0 | True | >=1| | three-up | setup | graph | setup txn - replay graph |
| >=1 |  False |  n/a | | two-up | graph | n/a | txn data |
| >=1 | True | 0 | | three-up | graph | setup | txn data - setup replay |
| >=1 | True | >=1 | | three-up | graph | graph | txn data - replay data |

### CODE REVIEW

**Comment by ryan953** [comment]
@JoshFerge not shown in the screen shots, but at the bottom of the email is a template to show three of the 'more error heavy' replays. the template should support the correct fields right now, but it's not in the mock data yet so ~untested.

File: src/sentry/features/__init__.py [hunk 1]
```diff
@@ -179,6 +179,7 @@
 default_manager.add("organizations:session-replay-sdk-errors-only", OrganizationFeature, FeatureHandlerStrategy.REMOTE)
 default_manager.add("organizations:session-replay-recording-scrubbing", OrganizationFeature, FeatureHandlerStrategy.INTERNAL)
 default_manager.add("organizations:session-replay-index-subquery", OrganizationFeature, FeatureHandlerStrategy.REMOTE)
+default_manager.add("organizations:session-replay-weekly-email", OrganizationFeature, FeatureHandlerStrategy.REMOTE)
```
Comment (JoshFerge) [inline src/sentry/features/__init__.py:182]: i'm guessing we can't use flagr for this, as it's too high volume. can use `INTERNAL` and we can specify an options based handler in getsentry

Comment (JoshFerge) [inline src/sentry/features/__init__.py:182]: can't use our remote handler for this as its too high volume. can use an `INTERNAL` strategy and we can specify one in getsentry

File: src/sentry/tasks/weekly_reports.py [hunk 2]
```diff
@@ -702,24 +724,34 @@ def render_template_context(ctx, user):
         user_projects = ctx.projects.values()
 
     has_issue_states = features.has("organizations:escalating-issues", ctx.organization)
+    has_replay_section = features.has("organizations:session-replay-weekly-email", ctx.organization)
```
Comment (scttcper) [inline src/sentry/tasks/weekly_reports.py:727]: add features.has('session-replay')

Comment (ryan953) [reply src/sentry/tasks/weekly_reports.py]: done!

**Review by scttcper — APPROVED** [review]

**Review by JoshFerge — APPROVED** [review]

**Pull request merged**